apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1635,7 +1635,7 @@ static void tc_driver_mtd_config_ops(void)
 	int fd;
 	int ret;
 	struct config_data_s config;
-	char *buf = "test";
+	char buf[8] = "test";
```

Comment:
Unnecessary change, I think

Reply:
Please revert file access type (10644 is correct)

Reply:
In another mcu, the string is allocated in the flash memory, and the pointer "char *buf" points to the readonly memeory. So in the test case "ret = ioctl(fd, CFGDIOC_GETCONFIG, &config);", an exception is popped up in memcpy() invoked by mtdconfig_readbytes() after the entry is found, because it tries to write data to readonly memory. 
It is better to define the buf as array allocated in stack to make sure the memory is writable and it will be safe for all mcus.

Reply:
File access type is fixed.